independent-projects/arc/pom.xml

```diff
@@ -69,11 +69,11 @@
         <module>processor</module>
         <module>tests</module>
 
-        <module>arquillian</module>
-        <module>atinject-tck-runner</module>
-        <module>cdi-tck-porting-pkg</module>
-        <module>cdi-tck-runner</module>
-        <module>lang-model-tck-runner</module>
+        <module>tck/arquillian</module>
+        <module>tck/atinject-tck-runner</module>
+        <module>tck/cdi-tck-porting-pkg</module>
+        <module>tck/cdi-tck-runner</module>
+        <module>tck/lang-model-tck-runner</module>
```

Comment:
Very minor but any chance you could name it `tcks` like the root `tcks` module? There are multiple of them and that would be more consistent?

Reply:
Done.